fapi-client/src/main/scala/com/gu/facia/api/models/collection.scala

```diff
@@ -192,18 +191,4 @@ object Collection {
     collection.copy(
       live = collection.live.filterNot(_.isSnap),
       draft = collection.draft.map(_.filterNot(_.isSnap)))}
-}
-
-case class Group(get: Int)
-
-object Group {
-  def fromGroups(groups: Groups): List[Group] = {
-    groups.groups.collect {
-      case IntegerString(n) => Group(n)
-    }
-  }
-
-  def toGroups(groups: List[Group]): Groups = {
```

Comment:
Was toGroups never used?

Reply:
It's a good point, I had the same thought.

Reply:
Looks like `fromGroups` is used (https://github.com/search?q=org%3Aguardian+fromGroups&type=code), but `toGroups` isn't (https://github.com/search?q=org%3Aguardian+toGroups&type=code).

Reply:
However I'd be inclined to have `toGroups` and `toGroupsJson` for feature parity in case anyone needs them? (less need for `toGroups` since it's being deprecated soon).